borg/archive.py

```diff
@@ -574,7 +574,7 @@ def process_file(self, path, st, cache, ignore_inode=False):
             status = 'A'  # regular file, added
         item = {
             b'path': safe_path,
-            b'link': st.st_nlink > 1,
+            b'link': st.st_nlink > 1,  # item is a hard link and has the chunks
```

Comment:
How about b'linkmaster' or b'mainlink'? Makes it clear that this about a link and it's somehow important / different...